File: public/codesamples/posts/graphes_grids/code1/Node.java
package posts.graphes_grids.code1;

import java.util.HashSet;
import java.util.Set;
import posts.graphes_grids.code2.Connection;

public class Node {
  private String name;
  private Set<Connection> connections = new HashSet<>();

  public Node(String name) {
    this.name = name;
  }

  public String getName() {
    return name;
  }

  public Set<Connection> getConnections() {
    return connections;
  }
}

File: public/codesamples/posts/graphes_grids/code1/Node.cs

using System.Collections.Generic;
using Posts.GraphesGrids.Code2;
namespace Posts.GraphesGrids.Code1
{
    public class Node
    {
        public string _name;
        private string Name
        {
            get { return Name; }
        }
        private readonly HashSet<Connection> _connections = new HashSet<Connection>();
        public HashSet<Connection> Connections
        {
            get { return _connections; }
        }
        public Node(string name)
        {
            _name = name;
        }
    }
}
